### PR TITLE
Created README.md file with code snippets for the main library functionalities.

### PR DESCRIPTION
**Pull request by ognian-baruh**
Signed-off-by: Ognian Baruh <fixed-term.Ognyan.Baruh@bosch.io>

### CODE REVIEW

**Comment by ognian-baruh** [comment]
@thjaeckle please don't merge this PR until a member of the team has approved it.

File: README.md
Comment (konstantina-gramatova) [inline README.md:62]: To keep the README crisp - this section (class implementation approach) could be just mentioned as a usage possibility with a reference to the full example but not as a whole section.

The key info in this example is actually the possibility to use an external Paho client.

File: README.md
Comment (konstantina-gramatova) [inline README.md:50]: In difference to the Go library, the connection configurations here are provided per connection not per client instance, thus, would be nice to have an example for using them - e.g. connecting to a specific port, using a specific keep_alive, etc. In this sense, the whole section below (using as a class) can be narrowed down to a config option mention.

File: README.md
Comment (konstantina-gramatova) [inline README.md:114]: Could be narrowed down and included in the connection configuration options in the beginning.

File: README.md
Comment (konstantina-gramatova) [inline README.md:295]: This is an example helper - please, leave it out.

**Review by konstantina-gramatova — CHANGES_REQUESTED** [review]

**Pull request merged**
